AgentInterfaces/SecretDetail.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Gaming.AgentInterfaces
+{
+    using System;
+
+    public class SecretDetail
+    {
+        /// <summary>
+        /// Identifier secret
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Secret Value
+        /// </summary>
+        public string Value { get; set; }
+
+        /// <summary>
+        /// The version of this secret in the internal Key Vault
+        /// </summary>
+        public string Version { get; set; }
+
+        /// <summary>
+        /// Expiration date of this secret
+        /// </summary>
+        public string ExpirationDate { get; set; }
+}
```

Comment:
there's a } missing, as depicted here [https://github.com/PlayFab/MpsAgent/actions/runs/6399628016/job/17371970994?pr=171](https://github.com/PlayFab/MpsAgent/actions/runs/6399628016/job/17371970994?pr=171)

Reply:
OMG yes. Don't know why I missed that